meal-planner-fredericton-ui/src/dashboard/Dashboard.jsx

```diff
@@ -220,11 +251,19 @@ const Dashboard = props => {
     </Paper>
   ) : null;
   
-
-
-
+  const [showDelete, setShowDelete] = React.useState(false);
+  const [currentRow, setCurrentRow] = React.useState(null);
+  
```

Comment:
Instead of open, setOpen I have named it as showDelete, setShowDelete for clear understanding. In order to know which row is selected I have created a separate state to track the currentRow which is the meal plan object.